ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -29,13 +29,20 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: installer-rehearse-aws
+- as: installer-rehearse-aws-private
```

Comment:
I think we do not have to change the test name here.

Reply:
I changed this name because I'm also working on https://github.com/openshift/release/pull/24888 and other features at this time, which are based on this PR, to avoid rebase conflicts and distinguish different types of clusters, I used different names. I will change them back later.